admin/src/main/java/org/rembx/jeeshop/admin/GatewayResource.java

```diff
@@ -0,0 +1,66 @@
+package org.rembx.jeeshop.admin;
+
+import org.apache.commons.io.IOUtils;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.CacheControl;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.StreamingOutput;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URLConnection;
+
+@Path("/")
+public class GatewayResource {
+
+    private static final String FALLBACK_RESOURCE = "/webapp/index.html";
+
+    @GET
+    @Path("/")
+    @Produces(MediaType.TEXT_HTML)
+    public Response getFrontendRoot() throws IOException {
+        return getFrontendStaticFile("index.html");
+    }
+
+    @GET
+    @Path("/{fileName:.+}")
+    public Response getFrontendStaticFile(@PathParam("fileName") String fileName) {
+        final InputStream requestedFileStream = GatewayResource.class.getResourceAsStream("/webapp/" + fileName);
```

Comment:
Provided filename could be a security issue here in case we put "../......."
Maybe we could use this standard directory instead:  https://quarkus.io/guides/http-reference#serving-static-resources

Reply:
done